src/components/Bar/Information.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { UserConsumer } from '../../MyContext'
+
+
+function Child(props) {
```

Comment:
What do you mean with child? this could have another name that gave more context on the purpose, also the short fn between lines 10-12 is unnecessary for any delivery, you should use console logs for debugging on development time only